docs/examples/irradiance-transposition/plot_rtranpose_year.py

```diff
@@ -0,0 +1,137 @@
+"""
+Reverse transposition using one year of hourly data
+===================================================
+
+With a brief look at accuracy and speed.
```

Comment:
I don't really see the point of this hanging sentence. Most visitors to this tutorial probably are more interested in the general concept - I suggest just nixing the line